osu.Game/Database/RealmBackedStore.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Platform;
+
+#nullable enable
+
+namespace osu.Game.Database
+{
+    public abstract class RealmBackedStore
```

Comment:
Not sure how necessary this class is for now; just following the previous structure we had in place.